gradle.properties

```diff
@@ -1,8 +1,8 @@
-projectVersion=1.0.0-SNAPSHOT
+projectVersion=1.1.0-SNAPSHOT
```

Comment:
There is no 1.0.0 release so this change is not necessary.

Reply:
Are you sure? https://search.maven.org/search?q=g:%22io.micronaut.pulsar%22%20AND%20a:%22micronaut-pulsar%22

Reply:
If that's the case, then the module was released in a wrong way. This is the only non-draft release:

![image](https://user-images.githubusercontent.com/559192/144190398-7f3b90bb-5d38-4694-8eef-91507f0c35e9.png)

And the version there is `1.0.0-SNAPSHOT`.

Now, on a second look I see this commit with the right version https://github.com/micronaut-projects/micronaut-pulsar/commit/78117558fb964b7e68a19eb59a43bc720baf921e but the tag is wrong https://github.com/micronaut-projects/micronaut-pulsar/releases/tag/v1.0.0-SNAPSHOT

I'll fix that.

So yes, we released 1.0.0 although it was hard to find in this repo.

@yawkat please ignore this comment and keep `1.1.0-SNAPSHOT` as the new version.

Reply:
@harissecic 1.0.0 is already in central so we can't roll it back, so the release would probably have to be 1.0.1 or 1.1.0 wil have to be the next version

Thanks for all your work on this module, looks really promising

Reply:
Thanks for the feedback. Happy to do something useful. Anyway, I'll need help understanding how to release 1.0.1 or 1.1.0 when the time comes, not to mess up tags again.